### DISCUSSION TITLE
Create Support for gemini models in playground

### DISCUSSION THREAD

**Discussion by anuraj-suman** [langfuse/langfuse#4019] [Ideas]
### Describe the feature or potential improvement

Currently we do not support Gemini models in playground in langfuse.
Please support this

**Comment by marcklingen** [comment]
Thanks for sharing this! @hassiebp do you think you can prioritize this some time soon to better support everyone on GCP?

**Comment by hassiebp** [comment]
Tracking progress here: https://github.com/langfuse/langfuse/issues/4028

**Comment by hassiebp** [comment]
We have just shipped Google Vertex AI support for playground and evals 👍🏾  Feel free to give it a spin and let us know your feedback! 🙏🏾 

https://langfuse.com/changelog/2024-11-28-google-vertex-ai-support-playground-evals

**Reply by marcklingen** [reply]
wohoo, thanks Hassieb!
